init-tools.sh

```diff
@@ -157,11 +157,11 @@ export ILASMCOMPILER_VERSION=$__ILASM_VERSION
 export NATIVE_TOOLS_RID=$__ILASM_RID
 
 echo "Initializing BuildTools..."
-echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR" >> $__init_tools_log
+echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR $__PACKAGES_DIR" >> $__init_tools_log
```

Comment:
Why is the __Packages_Dir needed here? I don't see it being used in your PR https://github.com/dotnet/buildtools/commit/b76c4b6e773f85efa9566b7a0090a8bca98cb57c#diff-519820c77125188495c6a22d71f859ce

Reply:
It is required so that the new compilers package get's now restored on the `corefx\packages\` directory so that the import of the props file works. Previously, it was being restored at the Tools directory and the import was only working by chance.

Reply:
If you check my BT PR, the changes on Roslyn.Common.props are the ones that are directly related to this change.